docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,41 @@
 title: Changelog
 ---
 
+## 12.14.0
+
+_Released 06/07/2023_
+
+**Features:**
+
+- A new testing type switcher has been added to the Spec Explorer to make it
```

Comment:
TBH this doesn't sound like a feature for our users, just an app change that will see.

Reply:
Do we have a definition of what we consider a feature?  This is providing new value to our users by surfacing the ability to switch testing types from the browser which is currently hidden behind clicking on the project name in the top left.

Reply:
For this issue, I have already deployed the code to NPM.  Just waiting to get this PR reviewed and pushed before moving the code to the `latest` tag. Is it okay to keep this as a feature for this release?